activemodel/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 ## Rails 5.2.0.beta2 (November 28, 2017) ##
 
-*   No changes.
+*   Fix validation callbacks on multiple context
```

Comment:
5.2.0.beta2 has already been released; new entries should be added to the very top of the file.

Reply:
Oh, I see. Thanks!